src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -32,7 +34,7 @@ namespace traits
 {
     /** Trait for int */
     template<>
-    struct PICToAdios<int>
```

Comment:
yes the problem is more if you go a system like tk1 which has a 32bit host, the compile suite will not cross-compile to it for testing.

nevertheless, I think it's fine as it is @psychocoderHPC